### ISSUE TITLE
yarn build fails

### ISSUE THREAD

**Issue by bytasv** [Semantic-Org/Semantic-UI-React#3211]
### Steps
1. Pull the latest repo version
2. `yarn install`
3. `yarn build`

### Expected Result
Build succeeds

### Actual Result
> [10:32:55] Error: Command failed: cross-env NODE_ENV=build-es babel /.../Semantic-UI-React/src -d /.../Semantic-UI-React/dist/es
Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel. (While processing preset: "/.../Semantic-UI-React/.babel-preset.js")

### Version
0.82.5

Anyone else experiencing this? I tracked down that it maybe because of `react-static` installing older version of webpack-cli, which in turn pulls in older `babel-core` with exact version that fails

**Comment by layershifter** [comment]
What OS are using? Build was successfully passed on Ubuntu recently, however there can be issue with Windows.

**Comment by bytasv** [comment]
@layershifter I'm running OS X 10.13.5 - could you also write down node, yarn and other versions that MAY potentially impact this?

**Comment by layershifter** [comment]
May be you have `babel` installed as global?

**Comment by bytasv** [comment]
@layershifter `babel` or `babel-cli` or some other specific package?

```
 ~/xxx   master ?  babel                                                                                             ✔  10731  13:44:43
zsh: command not found: babel
```

I assume not

**Comment by layershifter** [comment]
One of them. When you're typing `babel` in terminal and run this command, what you will receive?

**Comment by bytasv** [comment]
Both are not found. Do you know anyone on the team with Mac who could potentially test it?

**Comment by layershifter** [comment]
@bytasv can you please pull latest changes and try again? Don't forget about `yarn install` :)

**Comment by bytasv** [comment]
@layershifter any updates on this one?

**Comment by bytasv** [comment]
@levithomason sorry for pinging directly, but maybe you can take a look as well? This is kinda blocker not allowing to build :/

**Comment by layershifter** [comment]
@bytasv sorry, but I don't have any idea why it happens. I know that Levi uses Mac and we deployed the new release recently :thinking:

**Comment by levithomason** [comment]
I pulled the latest master (`79d5da8eb`), removed `node_modues`, ran `yarn`, then `yarn build` and all is well on my 2018 MacBook.  It was also working for me just before this commit SHA as well as on versions before 0.83.0.

It looks like you have a rogue babel `6.26.3` that is getting loaded from somewhere on your `PATH`, in your `node_modules`, or possibly in a directory above your project.  Here's the relavant portion of the error:

>Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel. (While processing preset: "/Users/vytautas/Projects/Semantic-UI-React/.babel-preset.js")

When our preset is called, the version loaded on your machine is incorrect for some reason.  Without the ability to repro, this leads me to believe it is a rogue babel version hiding on your machine somewhere.  Try these steps:

1. Ensure you have no babel on your path globally: `which babel` should return `babel not found`.
1. Remove any possible global modules: 
    ```
    $ npm uninstall babel @babel-core @babel-cli --global
    $ yarn global remove babel
    $ yarn global remove @babel-core
    $ yarn global remove @babel-cli
    ```
1. Nuke your `node_modules` in Semantic UI React: `rm -rf node_modules`
1. Reinstall: `yarn install`
1. Run a build: `yarn build`

Lastly, I would google around and punch stuff, eventually you'll find the answer and feel great.  Apologies your having difficulties with this.  Let us know if we've overlooked anything or if there is a repro and we'll gladly reopen this issue.  Best of luck 🙇
